### ISSUE TITLE
dot files are not ignored

### ISSUE THREAD

**Issue by TheMaverickProgrammer** [micromatch/glob-fs#11]
package: `"glob-fs": "^0.1.6"`

dot files (.git .gitignore ...) are included in my array results.

**Comment by mhulse** [comment]
Noticing this as well. `"version": "0.1.6"`.

**Comment by mhulse** [comment]
> TypeError: unsupported file type: undefined (file: /......../.DS_Store

**Comment by tunnckoCore** [comment]
Node versions? Cuz all was working, I believe.

**Comment by mhulse** [comment]
> Node versions? Cuz all was working, I believe.

Good point. I'm on `v6.7.0` (the latest stable at time of this comment), but there could be something up with my system that's different than most people's. Or, I'm using the code wrong.

 Sorry to raise any red flags if there are none. :octocat:

**Comment by tunnckoCore** [comment]
Yea it's possible, but as what I know, this package not uses so strange things to makes things not work between node version. So.. best will be to open issues and give code snippets and expected behaviors.
